docs/coding-style.md

```diff
@@ -39,8 +40,26 @@ Variable name `sut` (System Under Test) is recommended in unit tests that really
 ## Multi-line statements
 
 * Operators (&&, ||, +, :, ?, ?? and others) are placed at the beginning of a line.
-* Dots with invocation `.Method()` are placed at the beginning of a line.
-* Comma separating arguments is placed at the end of a line.
+* Dot inside an invocation `.Method()` is placed at the beginning of a line.
+* The comma separating arguments is placed at the end of a line.
+* Method declaration parameters should be on the same line. If S103 is violated, parameters should be placed each on a separate line; the first parameter should be on the same line with the declaration; the other parameters should be aligned with the first parameter.
+```
+public void MethodWithManyParameters(int firstParameter,
+                                     string secondParameter,
+                                     Function<int, string, string> complexParameter);
+```
```

Comment:
I intentionally didn't document this last time, because we said we need tooling first that we don't have. But with `should` it's probably OK-ish.

Reply:
I added it because from the last many PRs, I see this is the way we go.

Indeed if we'd have tooling in place, it shouldn't need to be documented.